src/Kovan-DssSpell.t.sol

```diff
@@ -169,197 +167,197 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for all system configuration changes
         //
         afterSpell = SystemValues({
-            pot_dsr: 1000000000000000000000000000,
-            pot_dsrPct: 0 * 1000,
-            vat_Line: 1216 * MILLION * RAD,
-            pause_delay: 60,
-            vow_wait: 3600,
-            vow_dump: 2 * WAD,
-            vow_sump: 50 * RAD,
-            vow_bump: 10 * RAD,
-            vow_hump: 500 * RAD,
-            cat_box: 10 * THOUSAND * RAD
+            dsr_rate:     0,               // In basis points
```

Comment:
I'd just rename it back to `pot_dsr` to keep the same naming convention of the rest of General system variables.